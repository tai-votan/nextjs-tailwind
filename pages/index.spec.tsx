import { render } from "@testing-library/react";
import Home from "./index";

describe("Homepage", () => {
  it("renders page", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
