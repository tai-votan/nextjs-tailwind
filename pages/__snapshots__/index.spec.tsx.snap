// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage renders page 1`] = `
<div>
  <div
    class="flex min-h-screen flex-col items-center justify-center py-2"
  >
    <main>
      common:title
    </main>
  </div>
</div>
`;
